### PR TITLE
[Feature][Connector-V2][Clickhouse] Add clickhouse connector timezone key,default UTC

### PR DESCRIPTION
**Pull request by gaopeng666**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/config/ClickhouseConfig.java [hunk 1]
```diff
@@ -75,6 +75,13 @@ public class ClickhouseConfig {
                     .noDefaultValue()
                     .withDescription("Clickhouse server password");
 
+    /** Clickhouse server timezone */
+    public static final Option<String> SERVER_TIME_ZONE =
+            Options.key("server_time_zone")
+                    .stringType()
+                    .defaultValue("UTC")
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/config/ClickhouseConfig.java]: I think the default value of `SERVER_TIME_ZONE` should be time zone on the machine which run SeaTunnel program.

Comment (gaopeng666) [reply seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/config/ClickhouseConfig.java]: need the system time zone or the JVM time zone? Why is the UTC time zone used by default through JDBC connection or MySQL cdc?

Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/config/ClickhouseConfig.java]: > Why is the UTC time zone used by default through JDBC connection or MySQL cdc?

@ic4y @hailin0 Can you tell us some information about this? Why use UTC not system time zone as default time zone?

Comment (gaopeng666) [reply seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/config/ClickhouseConfig.java]: I have set the timezone by default to the timezone of the seatunnel service

**Pull request closed without merging**